ui/apps/platform/src/Containers/Dashboard/Widgets/ComplianceLevelsByStandard.cy.jsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import { Router } from 'react-router-dom';
+import { ApolloProvider } from '@apollo/client';
+import { createBrowserHistory } from 'history';
+
+import configureApolloClient from 'configureApolloClient';
+import { standardEntityTypes } from 'constants/entityTypes';
+import { complianceBasePath, urlEntityListTypes } from 'routePaths';
+import ComplianceLevelsByStandard from './ComplianceLevelsByStandard';
+
+/*
+These standards have been formatted for easier verification of the expected ordering in
+tests compared to a direct hard coding in the mocked response below.
+
+id: [name, numFailing, numPassing]
+*/
+const standards = {
+    CIS_Kubernetes_v1_5: ['CIS Kubernetes v1.5', 8, 2],
+    HIPAA_164: ['HIPAA 164', 7, 3],
+    NIST_800_190: ['NIST SP 800-190', 6, 4],
+    NIST_SP_800_53_Rev_4: ['NIST SP 800-53', 5, 5],
+    PCI_DSS_3_2: ['PCI DSS 3.2.1', 4, 6],
+    'ocp4-cis': ['ocp4-cis', 3, 7],
+    'ocp4-cis-node': ['ocp4-cis-node', 2, 8],
+};
+
+const mock = {
+    data: {
+        controls: {
+            results: Object.entries(standards).map(([id, [, numFailing, numPassing]]) => ({
+                aggregationKeys: [{ id, scope: 'STANDARD' }],
+                numFailing,
+                numPassing,
+                numSkipped: 0,
+                unit: 'CONTROL',
+            })),
+        },
+        complianceStandards: Object.entries(standards).map(([id, [name]]) => ({
+            id,
+            name,
+        })),
+    },
+};
+
+beforeEach(() => {
+    localStorage.clear();
+});
+
+const setup = () => {
+    const history = createBrowserHistory();
+
+    cy.intercept('POST', '/api/graphql?opname=getAggregatedResults', (req) => {
+        req.reply(mock);
+    });
+
+    cy.mount(
+        <Router history={history}>
+            <ApolloProvider client={configureApolloClient()}>
+                <ComplianceLevelsByStandard />
+            </ApolloProvider>
+        </Router>
+    );
+};
+
+describe('<ComplianceLevelsByStandard />', () => {
```

Comment:
1. What do you think about `'Dashboard ComplianceLevelsByStandard'`  for similarity with other descriptions and to remove issue of sort order for leading punctuation?
2. Also can you:
     * add picture of intentionally provoked local failure
     * verify correct report for intentionally provoked CI failure

Reply:
What do you think about doing something like:
`describe(Cypress.spec.relative, () => {`

This would give the full path to the component under test in the logs, and in this case would be the equivalent of:
`describe('src/Containers/Dashboard/Widgets/ComplianceLevelsByStandard.cy.jsx', () =>`